data/mappings/info_rules.json

```diff
@@ -20,12 +20,12 @@
     "MOUSE_SHARED_EP": {"info_key": "usb.shared_endpoint.mouse", "value_type": "bool"},
     "LAYOUTS": {"info_key": "community_layouts", "value_type": "list"},
     "LED_MATRIX_DRIVER": {"info_key": "led_matrix.driver"},
-    "RGB_MATRIX_DRIVER": {"info_key": "rgb_matrix.driver"},
     "LTO_ENABLE": {"info_key": "build.lto", "value_type": "bool"},
     "MCU": {"info_key": "processor", "warn_duplicate": false},
     "MOUSEKEY_ENABLE": {"info_key": "mouse_key.enabled", "value_type": "bool"},
     "NO_USB_STARTUP_CHECK": {"info_key": "usb.no_startup_check", "value_type": "bool"},
     "PIN_COMPATIBLE": {"info_key": "pin_compatible"},
+    "RGB_MATRIX_DRIVER": {"info_key": "rgb_matrix.driver"},
```

Comment:
This core edit should be excluded from the PR.

Reply:
That's fair! It's only in there because it was discovered while I was trying to add data driven stuff to the JSON; I wondered why it was not working (defining RGB_MATRIX_DRIVER didn't change behaviour), and it turned out it wasn't actually mapped in the core file. A maintainer kindly told me what to add to this file, so I did that, but the same maintainer also added the same exact line to a different place in the file (I put it in alphabetical order because everything else in this file is also alphabetically sorted, but the maintainer put it in a different place, probably just because they added it very very quickly and submitted a PR to merge it within minutes of the issue being noticed). So all this part is doing is just correcting the alphabetical order of this file, nothing more.

It can be excluded if required or split out into a separate (trivial) PR, it makes no functional difference other than being potentially confusing / easier to miss if you're looking for it because it's in the wrong place.

Reply:
It should definitely be removed from this PR.

Reply:
Will do then!